core/util/src/main/scala/net/liftweb/util/HttpHelpers.scala

```diff
@@ -138,7 +138,7 @@ trait HttpHelpers {
   */
   def toHashMap[A,B](in : Map[A,B]) : HashMap[A,B] = {
     val ret = new HashMap[A,B];
-    in.keysIterator.foreach { k => ret += Pair(k, in(k)) }
+    in.keysIterator.foreach { k => ret += ((k, in(k))) }
```

Comment:
That's what I first tried thinking it should work but no, scalac complained about it, I can't remember what the error was, I imagine that to eliminate the extra parenthesis I would have to use this syntax: `ret.+=(k, in(k)`  or replace `Pair` for `Tuple2`